canvas_modules/harness/test_resources/parameterDefs/toggle_paramDef.json

```diff
@@ -149,7 +149,7 @@
       {
         "parameter_ref": "toggle_warning",
         "label": {
-          "default": "Warning"
+          "default": "toggle Warning"
```

Comment:
One last change, can you make sure the corresponding paramDef under the tests folder has the same change?